s1m0nWasTaken/CS4337_BigDataProject
# QOL additons; formatter and pr template

## Summary

add a /docs folder  with a:
 - pr template
 - brief explanation of the spotless formatter

add spotless formatter to the gradle.build.  Enforces googles java style
formatted exising java files

## How to verify

 - pull the branch
 - mess up the formatting
 - try to `./gradlew build`
 - when that fails `./gradlew spotlessApply`
 - `./gradlew build` again, it should work this time
 - then see that the fromatting in the files have been changed
